src/Agent/NewRelic/Agent/Core/Configuration/DefaultConfiguration.cs

```diff
@@ -1135,53 +1137,25 @@ public TimeSpan ErrorEventsHarvestCycle
         }
 
         public virtual uint ErrorsMaximumPerPeriod { get { return 20; } }
-        public virtual IEnumerable<string> ExceptionsToIgnore { get { return ServerOverrides(_serverConfiguration.RpmConfig.ErrorCollectorErrorsToIgnore, _localConfiguration.errorCollector.ignoreErrors.exception); } }
 
-        private void ParseExpectedErrorConfigurations()
+        private IEnumerable<string> _exceptionsToIgnore;
+        public virtual IEnumerable<string> ExceptionsToIgnore
```

Comment:
We should consider parsing this config setting in the same place that we parse the other configuration related to ignored errors.

Reply:
I'm planning to remove this configuration and replace its usages with the `IgnoreErrorConfiguration` in the next work. I still keep it here because I don't want to modify existing tests that use it.

Reply:
This setting uses a different local and server-side config setting than the other ignore settings that were just added. My assumption is that we still need to fall back and use these settings if the other settings are not provided?

Reply:
Yea you are right, even though I did combine the old config with new config in [here](https://github.com/newrelic/newrelic-dotnet-agent/blob/fed4fe4f720e17a884f40968e0f688aaed1d34b6/src/Agent/NewRelic/Agent/Core/Configuration/DefaultConfiguration.cs#L2077), but I forgot that we still have to parse it.

Reply:
I just added to code to parse the old config in the same place of the new config parser. I still keep the `ExceptionToIgnore` coz I don’t want to modify some tests just yet. I will remove it in the next work.